django/db/models/lookups.py

```diff
@@ -122,7 +122,7 @@ def process_rhs(self, compiler, connection):
             # Ensure expression is wrapped in parentheses to respect operator
             # precedence but avoid double wrapping as it can be misinterpreted
             # on some backends (e.g. subqueries on SQLite).
-            if sql and sql[0] != "(":
+            if not isinstance(value, Value) and sql and sql[0] != "(":
```

Comment:
This _hack_ is getting even more ugly but I don't see a way around it without introducing another hook to `Expression` for now.

Reply:
Yeah, unfortunately :shrug: